File: src/main/java/cz/vhromada/catalog/gui/common/StatsTableCellRenderer.java
package cz.vhromada.catalog.gui.common;

import javax.swing.table.DefaultTableCellRenderer;

/**
 * A class represents table cell renderer for changing alignment of the cell.
 *
 * @author Vladimir Hromada
 */
public class StatsTableCellRenderer extends DefaultTableCellRenderer {

    /**
     * SerialVersionUID
     */
    private static final long serialVersionUID = 1L;

    /**
     * Creates a new instance of StatsTableCellRenderer.
     */
    public StatsTableCellRenderer() {
        setVerticalAlignment(CENTER);
        setHorizontalAlignment(CENTER);
    }

}

File: src/main/kotlin/cz/vhromada/catalog/gui/common/StatsTableCellRenderer.kt
package cz.vhromada.catalog.gui.common

import javax.swing.SwingConstants
import javax.swing.table.DefaultTableCellRenderer

/**
 * A class represents table cell renderer for changing alignment of the cell.
 *
 * @author Vladimir Hromada
 */
class StatsTableCellRenderer : DefaultTableCellRenderer() {

    init {
        verticalAlignment = SwingConstants.CENTER
        horizontalAlignment = SwingConstants.CENTER
    }

}
